test/performance/odp_l2fwd.c

```diff
@@ -1571,6 +1580,8 @@ static void parse_args(int argc, char *argv[], appl_args_t *appl_args)
 	appl_args->chksum = 0; /* don't use checksum offload by default */
 	appl_args->pool_per_if = 0;
 	appl_args->num_pkt = 0;
+	appl_args->packet_len = POOL_PKT_LEN;
+	appl_args->seg_len = POOL_SEG_LEN;
```

Comment:
Sorry, I should have realized earlier that segment length should probably default to packet length, if the segment length argument is not given.

Reply:
ACK